sources/tools/Stride.FreeImage/Stride.FreeImage.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project>
+    <Import Project="..\..\targets\Stride.props" />
+    <PropertyGroup>
+        <TargetFramework>$(StrideXplatEditorTargetFramework)</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+    <ItemGroup>  
+        <StrideNativeLib Include="$(MSBuildThisFileDirectory)..\..\..\deps\FreeImage\Release\**\*.dll">
```

Comment:
The same link in the original project should probably be removed.

Reply:
Thanks I forgot about that, fixed!